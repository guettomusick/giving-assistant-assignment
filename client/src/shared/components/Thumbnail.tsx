import styled from 'styled-components';

const Thumbnail = styled.img`

`;

export default Thumbnail;