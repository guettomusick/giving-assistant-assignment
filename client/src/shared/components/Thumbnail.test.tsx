import React from 'react';
import renderer from 'react-test-renderer';

import Thumbnail from './Thumbnail';

describe('Thumbnail', () => {
  it('renders', () => {
    const component = renderer.create(
      <Thumbnail src='test.jpg' />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
