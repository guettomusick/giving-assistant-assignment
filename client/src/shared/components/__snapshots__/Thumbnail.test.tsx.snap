// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail renders 1`] = `
<img
  className="sc-AxjAm dbBgCN"
  src="test.jpg"
/>
`;
